consensus/istanbul/backend/engine.go

```diff
@@ -453,10 +453,15 @@ func (sb *backend) Finalize(chain consensus.ChainReader, header *types.Header, s
 	}
 
 	var rewardSpec *reward.RewardSpec
-	var err error
+
+	rules := chain.Config().Rules(header.Number)
+	pset, err := sb.governance.ParamsAt(header.Number.Uint64())
```

Comment:
maybe pSet? parameter set?

Reply:
Let's fix it later.

Reply:
Other codes use pset as well, so we'll update to update them all...